A3A/addons/ultimate/config.cpp

```diff
@@ -176,4 +192,59 @@ class Extended_PreInit_EventHandlers
 	{
         init = "call A3U_fnc_init";
     };
-};
\ No newline at end of file
+};
+class CfgSounds
+{
+	class CRAMALARM
+	{
+		name = "CRAM_Alarm";
+		sound[] = {"functions\D37_cram\Sound\CRAM_ALARM.ogg", 1.0, 1.0};
+		titles[] = {0, ""};
+	};
+};
```

Comment:
This is duplicated like 3 times, putting it in 3 places won't make it magically work